processing/src/main/java/io/druid/query/aggregation/post/ExpressionPostAggregator.java

```diff
@@ -70,22 +76,33 @@ public ExpressionPostAggregator(
       @JacksonInject ExprMacroTable macroTable
   )
   {
+    this(name, expression, ordering, macroTable, ImmutableMap.of());
+  }
+
+  /**
+   * Constructor for {@link #decorate(Map)}.
+   */
+  private ExpressionPostAggregator(
+      final String name,
+      final String expression,
+      @Nullable final String ordering,
+      final ExprMacroTable macroTable,
+      final Map<String, Function<Object, Object>> finalizers
```

Comment:
To do this, I'd have to change the `decorate` method to use ImmutableMap builders rather than streams. Instead, I just removed the `ImmutableMap.copyOf` in the constructor, since it's not really necessary. Nothing would be trying to modify the finalizers map anyway.